classes/task/look_for_updates.php

```diff
@@ -46,7 +48,41 @@ public function execute() {
             $core->fetchLibrariesMetadata(!$extcom);
             set_config('external_communication_notify', $extcom ? false : time(), 'mod_hvp');
 
-            // TODO: Notify admin if there's an update available!
+            // Notify admin if there are updates available!
+            $update_available = \get_config('mod_hvp', 'update_available');
+            $current_update = \get_config('mod_hvp', 'current_update');
+            $admin_notified = \get_config('mod_hvp', 'admin_notified');
+            if ($admin_notified !== $update_available &&  // Admin has not been notified about this update
+                $update_available !== false &&
+                $current_update !== false &&
+                $current_update < $update_available) { // New update is available
+
+                // Send message
+                $updatesurl = new \moodle_url('/mod/hvp/library_list.php');
+                $message = new \stdClass();
+                $message->component         = 'mod_hvp';
+                $message->name              = 'updates';
+                $message->userfrom          = $USER;
+                $message->userto            = get_admin();
+                $message->subject           = get_string('updatesavailabletitle', 'mod_hvp');
+                $message->fullmessage       = get_string('updatesavailablemsgpt1', 'mod_hvp') . ' ' .
+                                              get_string('updatesavailablemsgpt2', 'mod_hvp') . "\n\n" .
+                                              get_string('updatesavailablemsgpt3', 'mod_hvp', date('Y-m-d', $update_available)) . "\n" .
+                                              get_string('updatesavailablemsgpt4', 'mod_hvp', date('Y-m-d', $current_update)) . "\n\n" .
+                                              $updatesurl;
+                $message->fullmessageformat = FORMAT_PLAIN;
+                $message->fullmessagehtml   = '<p>' . get_string('updatesavailablemsgpt1', 'mod_hvp') . '<br/>' .
+                                              get_string('updatesavailablemsgpt2', 'mod_hvp') . '</p>' .
+                                              '<p>' . get_string('updatesavailablemsgpt3', 'mod_hvp', '<b>' . date('Y-m-d', $update_available) . '</b>') . '<br/>' .
+                                              get_string('updatesavailablemsgpt4', 'mod_hvp', '<b>' . date('Y-m-d', $current_update) . '</b>') . '</p>' .
+                                              '<a href="' . $updatesurl .  '" target="_blank">' . $updatesurl . '</a>';
+                $message->smallmessage      = '';
+                $message->notification      = 1;
+                message_send($message);
```

Comment:
I'm wondering if this is up to the one setting up Moodle. I think you can request to have the messages show up as a popups instead.